src/diagnostics/MOM_harmonic_analysis.F90

```diff
@@ -1,7 +1,8 @@
 !> Inline harmonic analysis (conventional)
 module MOM_harmonic_analysis
 
-use MOM_time_manager,  only : time_type, real_to_time, time_type_to_real, get_date, increment_date, &
+use MOM_time_manager,  only : time_type, real_to_time, time_type_to_real, &
+                              set_date, get_date, increment_date, &
```

Comment:
It is the practice elsewhere in the MOM6 code not to use continuation lines for the module use statements, but rather to list the module that is being used appear on every line.  It is legal Fortran either way, but listing the modules in on each line is helpful when grepping through the code to figure out the module dependencies.